sqlserver/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ***Changed***:
 
+* Remove python 2 references from SQL Server integration ([#15606](https://github.com/DataDog/integrations-core/pull/15606))
+
+***Changed***:
```

Comment:
One more here. Would you mind deleting this line and move the entry under the Changed header in line 5?

Reply:
👍 fixed